src/Components/Pages/TransferPage.tsx

```diff
@@ -516,19 +562,44 @@ const TransferPage = () => {
               }
             />
             <section>
-              <Button type="submit" fullsize variant="primary">
-                Start transfer
+              <Button
+                type="submit"
+                fullsize
+                variant="primary"
+                className={classes.transferButton}
+              >
+                Start Transfer
               </Button>
             </section>
-            {/* <section>
-            <QuestionCircleSvg
-              onClick={() => setAboutOpen(true)}
-              className={classes.faqButton}
-            />
-          </section> */}
           </Form>
         )}
       </Formik>
+      <section className={classes.footer}>
+        <NavLink
+          style={{ textDecoration: "none" }}
+          className={classes.footerText}
+          to={{ pathname: "https://cere.network/" }}
+          target="_blank"
+        >
+          Cere Homepage
+        </NavLink>
+        <NavLink
+          style={{ textDecoration: "none" }}
+          className={classes.footerText}
+          to={{ pathname: "https://explorer.cere.network" }}
```

Comment:
Link should point to the Cere Mainnet rpc node, staking tab.

Reply:
Updated